internal/support/notifications/rest_subscription.go

```diff
@@ -284,6 +298,20 @@ func splitVars(vars string) []string {
 	return strings.Split(vars, ",")
 }
 
+func validateEmailAddresses(s models.Subscription) error {
+	err := errors.NewErrMailAddrContainsCRLF(s.Slug)
+	for _, c := range s.Channels {
+		if c.Type == models.ChannelType(models.Email) {
+			for _, m := range c.MailAddresses {
+				if strings.ContainsAny(m, "\n\r") {
+					return err
```

Comment:
Could we return an error with all the email addresses which have failed the validation check. I think that would be helpful to both the client and during any troubleshooting that might take place while EdgeX is in use. For example, I created a notification with the following payload:
```
{
	"slug": "sys-admin3",
	"receiver": "System Administrator",
	"subscribedCategories": [
		"HW_HEALTH"
	],
	"subscribedLabels": [
		"temperature",
		"humidity"
    ],
	"channels": [
		{
			"type": "EMAIL",
			"mailAddresses": [
				"AnthonyMBonafide@gmail.com\r\n",
				"jack@example.com\n\r"
	    	]
		}
	]
}
```

and received the response body:

```
Subscription 'sys-admin3' mail addresses contain CRLF
```

Where a list of the emails would be nice. Maybe something like:

```
Subscription 'sys-admin3' mail addresses contain CRLF: [jack@example.com,...]
```

Reply:
Yes, makes sense, will do.